### ISSUE TITLE
Question about the naive baselines

### ISSUE THREAD

**Issue by klauscc** [gqk/LAE#1]
Hi authors,
Congratulations on your great work! I have a few questions about you paper. It would be great if you can kindly answer them!

In your paper Tab.1 and Tab. 2, your baselines are extremely high, i.e. comparable or even better then L2P and DualPrompt.
Also in Tab.1 and Tab.2, the results of Seq-FT are also much higher the numbers reported in L2P paper. For CIFAR100 dataset, L2P reported 33.61% for Seq-FT while you reported 77.61%.
Could you explain why this happened? Did you use the task identity during inference? I.e., for the test set in each task, did you filter out the logits of other tasks? 

Also is my understanding to the naive baseline correct? For CIFAR-100, you insert some new parameters into the pretrained model, i.e. 20 prompts and a 100-class classifier. Then you train the new parameters sequentially on each task. During inference, given a test image, you predict the category from the 100-class classifier without any other information (i.e. task class mask). Is this right?

Thanks!

**Comment by gqk** [comment]
Hi klauscc,

I'm very sorry for the delayed response as I've been quite busy lately.

Thank you for your interest in our work.

Regarding Seq-FT, we use the same training and inference method as DualPrompt, except for the use of SGD optimizer, learning rate of 0.01 (scale based on batch size 128), and weight decay of 5e-4, since we fine-tune all parameters of the pre-trained model. The difference in performance could be attributed to the local classification loss.

Regarding the naive baseline, your understanding is basically correct. We did insert the prefixes with length 20 into each of the shallowest 5 transformer blocks and used the same training and inference method as DualPrompt. You can verify this using the code released by [Jaeho Lee](https://github.com/JH-LEE-KR) or the DualPrompt authors.

If you have any further questions, please don't hesitate to reach out to me. And if I don't reply promptly, please feel free to send me an email (gqk at stu.pku.edu.cn) as a reminder.
 
Thank you again for your interest.

Best regards,

Qiankun Gao

**Comment by gqk** [comment]
Hello @klauscc,

I noticed that there hasn't been any progress on the issue for quite a long time, so I will go ahead and close this issue. If you have any other questions or concerns, please don't hesitate to reach out to me again.

Best regards,

Qiankun Gao
